### DISCUSSION TITLE
Lost my account

### DISCUSSION THREAD

**Discussion by CamilaBernardino** [community/community#120415] [GitHub Education]
### Hi everyone,

I lost my previous account CamilaDominika, because I changed my cell phone and therefore I can't verify the account with two-factor authentication, but now I can't apply the student benefit to my new account, since I had put it in the account I lost

**Comment by ettaboyle** [comment]
The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
